backend/plugins/frontline_api/src/modules/reports/graphql/reportQueries.ts

```
@@ -336,12 +359,23 @@ export const reportQueries = {
     });
 
     pipeline.push(
+      {
+        $addFields: {
+          closedAtDate: {
+            $cond: {
+              if: { $eq: [{ $type: '$closedAt' }, 'string'] },
+              then: { $dateFromString: { dateString: '$closedAt' } },
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)